package.json

```diff
@@ -19,7 +19,7 @@
     "node": ">=0.8"
   },
   "dependencies": {
-    "backbone": "1.1.2",
+    "backbone": "1.2.3",
```

Comment:
Ha, @SimenB I figured you'd look it up and prove me wrong. Definitely felt like we locked it a *long* time ago since Backbone breaking all the time.